spec.md

```diff
@@ -112,6 +113,16 @@ This header allows brokers to reject requests from marketplaces for versions the
 The marketplace MUST authenticate with the service broker using HTTP
 basic authentication (the `Authorization:` header) on every request. The broker is responsible for validating the username and password and returning a `401 Unauthorized` message if credentials are invalid. It is RECOMMENDED that brokers support secure communication from platform marketplaces over TLS.
 
+## API Routes
+
+All endpoint routes in the Service Broker API are defined as URL suffixes. The Service Broker API MAY be exposed from a common URL prefix. An individual broker MAY also choose to expose multiple Service Broker APIs from different URL prefixes, but each MUST have a unique URL prefix.
```

Comment:
Perhaps a slight rewording to make it clear that they can't change prefixes across APIs:
```
All endpoint routes defined in this specification are to be interpreted as URL suffixes.
When a broker is registered with a platform, and provides a URL prefix, all supported
endpoints MUST be based on that URL prefix. In other words, within the scope of a
single broker's use for a platform, the URL prefix MUST NOT change.
```
I think we can stop there, but if people want to be very explicit we could add something like:
```
While a broker MAY be registered to multiple platforms, or even if it is registered to the
same platform more than once,  each registration MUST be treated as a different broker
for the purpose of the URL prefix/suffix scoping.
```